team-B-description.md

```diff
@@ -0,0 +1,11 @@
+### E팀 소개
+
+#### 팀 구성: 여종현, 서민경, 이태우, 강성일
+#### 팀장: 여종현
+
+
+#### 구성원 소개 
+이름:여종현   
+관심 컴포넌트: backend.ai-maanger, backend.ai-agent, backend.ai-cli
```

Comment:
Typo error found in here. Please update `backend.ai-maanger` to `backend.ai-manager`

Reply:
I've just updated it. Thanksn